File: baekjoon/15449/Main.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
      
        String[] arr_1 = br.readLine().split(" "); 
        int[] arr = new int[5];

        for(int i=0; i<5; i++){
            arr[i] = Integer.parseInt(arr_1[i]);
        }
        Arrays.sort(arr);
        int ans = 0;

        if(arr[4] < arr[3]+arr[2]){
            ans++;
        }
        if(arr[4] < arr[3]+arr[1]){
            ans++;
        }
        if(arr[4] < arr[3]+arr[0]){
            ans++;
        }
        if(arr[4] < arr[2]+arr[1]){
            ans++;
        }
        if(arr[4] < arr[2]+arr[0]){
            ans++;
        }
        if(arr[4] < arr[1]+arr[0]){
            ans++;
        }
        if(arr[3] < arr[2]+arr[1]){
            ans++;
        }
        if(arr[3] < arr[0]+arr[2]){
            ans++;
        }
        if(arr[3] < arr[1]+arr[0]){
            ans++;
        }
        if(arr[2] < arr[1]+arr[0]){
            ans++;
        }
        System.out.println(ans);
        
   
    }
}

File: baekjoon/15449/Main.c
#include <stdio.h>

int main(void) {
    int a[5];
		int ans = 0;

    for (int i=0; i<5; i++){
			scanf("%d", &a[i]);
		}

    for (int i=0; i<4; i++) {
        for (int j=0; j<4-i; j++) {
            if (a[j]>a[j+1]) {
                int t=a[j];
                a[j]=a[j+1];
                a[j+1]=t;
            }
        }
    }// 정렬

    for (int i=0; i<5; i++) {
        for (int j=i+1; j<5; j++) {
            for (int k=j+1; k<5; k++) {
                if (a[i]+a[j]>a[k]) ans++;
            }
        }
    } // 갱신


    printf("%d", ans);
}